core/client/grpcc/grpc_client.go

```diff
@@ -230,9 +228,9 @@ func (c *Client) ListHistory(ctx context.Context, userID, appID string, start, e
 
 // Update creates an UpdateEntryRequest for a user, attempt to submit it multiple
 // times depending on RetryCount.
-func (c *Client) Update(ctx context.Context, userID, appID string, profileData []byte,
+func (c *Client) Update(ctx context.Context, appID, userID string, profileData []byte,
```

Comment:
hope you caught all the call-sites of this change!

Reply:
I think I did... the unit tests helped me track down places where I had missed it.